src/server/components/lists/__tests__/controllers.test.ts

```diff
@@ -119,26 +111,9 @@ describe("Lists Controllers", () => {
   }
 
   function spySendApplicationConfirmationEmail(): any {
-    return jest
-      .spyOn(notify, "sendApplicationConfirmationEmail")
-      .mockResolvedValue(true);
+    return jest.spyOn(notify, "sendApplicationConfirmationEmail").mockResolvedValue(true);
   }
 
-  describe("listsGetController", () => {
-    test("it renders question page when serviceType is undefined", () => {
-      listsGetController(req, res);
-      req.params.page = "";
-
-      expect(res.render).toHaveBeenCalledWith("lists/question-page", {
-        ...DEFAULT_VIEW_PROPS,
-        ...{ ...req.params, ...req.query, ...req.body },
-        partialToRender: "question-service-type.njk",
-        csrfToken: "",
-        getServiceLabel,
-      });
-    });
-  });
```

Comment:
Is can see why this specific test is no longer necessary. But is there a chance user could edit the url and type a service that doesn't exist?

If so what page would the user just see a 404 page?

Reply:
They should see a 404 (page not found), but just checked and it's a 500. `src/server/components/lists/find/handlers/params/serviceType.ts` fixed by changing the schema to .only().

Reply:
https://github.com/UKForeignOffice/lists/pull/654/commits/bc1012500fc078b8ee39a61d4a6abe7cec2762d0

I've resolved the issue, and written tests for them on a unit level and integration level (supertest).